.github/workflows/docs-pull-requests.yaml

```
@@ -1,76 +0,0 @@
-name: Fondue Documentation CI
-
-on:
-    pull_request:
-
-# Ensures that only one workflow per branch will run at a time.
-concurrency:
-    group: ${{ github.workflow }}-${{ github.ref }}
-    cancel-in-progress: true
-
-jobs:
-    lint:
-        name: Lint
-        runs-on: ubuntu-latest
-        timeout-minutes: 10
-
-        steps:
-            - name: Checkout default branch
-              uses: actions/checkout@v4
-
-            - name: Use pnpm
-              uses: pnpm/action-setup@v2
-              with:
-                  version: latest
-                  run_install: false
-
-            - name: Use Node.js
-              uses: actions/setup-node@v4
-              with:
-                  node-version-file: ".nvmrc"
-                  cache: "pnpm"
-
-            - name: Install npm dependencies
-              run: pnpm i --frozen-lockfile
-
-            - name: Generate React icons
-              run: pnpm --stream --filter {packages/icons} build:generate-react-icons
-
-            - name: Contentlayer build
-              run: pnpm --stream --filter {docs} build:contentlayer
-
-            - name: Lint code
-              run: pnpm --stream --filter {docs} lint
-
-    typecheck:
-        name: Typecheck
-        runs-on: ubuntu-latest
-        timeout-minutes: 10
-
-        steps:
-            - name: Checkout default branch
-              uses: actions/checkout@v4
-
-            - name: Use pnpm
-              uses: pnpm/action-setup@v2
-              with:
-                  version: latest
-                  run_install: false
-
-            - name: Use Node.js
-              uses: actions/setup-node@v4
-              with:
-                  node-version-file: ".nvmrc"
-                  cache: "pnpm"
-
-            - name: Install npm dependencies
-              run: pnpm i --frozen-lockfile
-
-            - name: Generate React icons
-              run: pnpm --stream --filter {packages/icons} build:generate-react-icons
-
-            - name: Contentlayer build
-              run: pnpm --stream --filter {docs} build:contentlayer
-
-            - name: Typecheck code
-              run: pnpm --stream --filter {docs} typecheck
```

I think as part of our CI we should build the two substorybook to catch issues? Otherwise we only rely on manual testing of opening all 3 storybooks and check that things are fine.

```suggestion
- name : Build Storybook Packages
  run: pnpm --stream --filter {packages/fondue} --stream --filter {packages/ui}  build:storybook
- name: Build Storybook Docs
  run: pnpm build:storybook
```

---

i added the build storybook to the ci pipeline